updating/updating-restricted-network-cluster/mirroring-image-repository.adoc

```diff
@@ -0,0 +1,169 @@
+:_content-type: ASSEMBLY
+[id="mirroring-ocp-image-repository"]
+= Mirroring the {product-title} image repository
+include::_attributes/common-attributes.adoc[]
+:context: mirroring-ocp-image-repository
+
+toc::[]
+
+You must mirror container images onto a mirror registry before you can update a cluster in a disconnected environment. You can also use this procedure in connected environments to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+[NOTE]
+====
+This registry must be running at all times as long as the cluster is running.
+====
+
+There are two methods for mirroring images onto a mirror registry:
+
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-ocp-resources-ocmirror[Using the oc-mirror OpenShift CLI (`oc`) plugin]
+
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#update-mirror-repository-adm-release-mirror_mirroring-ocp-image-repository[Using the `oc adm release mirror` command]
+
+Compared to using `oc adm release mirror`, the oc-mirror plugin has the following advantages:
+
+* It can mirror content other than container images.
+
+* After mirroring images for the first time, it is easier to update images in the registry.
+
+* oc-mirror provides an automated way to mirror the release payload from Quay, and also builds the latest graph-data image for the OpenShift Update Service running in the disconnected environment.
```

Comment:
@jboxman-rh I added this one line after your peer review, would you mind checking it out to see if it looks good to you?

Reply:
@skopacz1 LGTM